examples/src/providers/rule_closures/transformers.js

```diff
@@ -4,6 +4,43 @@ const { ICoreHttpError } = require('rule-harvester');
 // These are example closures to transform the facts object
 
 module.exports = [
+  {
+    /**
+     * reformat-udp-message
+     * Reformat an incoming UDP Message by pulling all the message "properties"
+     * into the root of the facts object. Why? So we can use the same rules we use for the Directory Watcher Input.
+     * @param - facts
+     * @param - context
+     * @return - facts, updated.
+     **/
+    name: 'reformat-udp-message',
+    async handler(facts, context) {
+      // The UDP Input gives us a special object. Check to see if this facts object is an AMQP input.
```

Comment:
I don't understand the reference to AMQP Input here. Can you reword/clariffy?

Reply:
Yeah...thats what I get for copying and pasting :)